### DISCUSSION TITLE
Sensor and PCAP Importing not working after following Security Onion Essentials videos?

### DISCUSSION THREAD

**Discussion by johndabomb44** [Security-Onion-Solutions/securityonion#2302] [Unsupported Versions]
Hello!

I followed the steps outlined in the Security Onion Essentials training course on YouTube.

I'm not sure my IDS is working correctly as when I ran "sudo so-test", the SOC dashboard did not populate with a lot of alerts like the one used in the training videos. I also tried importing pcaps from malware-traffic-analysis and I am not getting any kinds of alerts.

The following is my setup:

Virtualization Platform = VirtualBox

Network Adapter 1 = enp0s3 (set to Bridged Adapter, confirmed working as I successfully can reach the SOC web interface and use "wget" to pull pcaps from malware-traffic-analysis.net)

Network Adapter 2 = enp0s8 (set to Internal Network, Promiscious Mode, Allow All per the instructions at https://docs.securityonion.net/en/2.3/virtualbox.html?highlight=virtualbox#virtualbox)

Install Mode = Eval

enp0s3 = 192.168.1.35 (Management)
enp0s8 = Monitor

Grafana,OSQuery,Wazuh,TheHive,Playbook,Strelka all enabled

Have I missed something or was the example used in the YouTube video populated with other traffic outside of what is provided in "so-test"?

**Comment by weslambert** [comment]
You might want to first try checking the monitor NIC to ensure the packets that so-test are replaying are hitting the interface.  Is this the case?

**Reply by johndabomb44** [reply]
The command works as I get 55304 amount of successful packets replayed.

I noticed the packets get replayed to a "bond0" interface which ifconfig says it's a promiscuous master to my enp0s8 interface which I personally selected to be my monitor interface (ifconfig says enp0s8 is the promiscuous slave so I'm guessing this is all correct?).

If this is not supposed to be happening, I don't know what I have goofed on install or if I'm supposed to do something other than what was in the training video.
